packages/workbench/src/routes/catalog/$contract/$key.tsx

```diff
@@ -67,7 +79,7 @@ function Inspector(props: {
 
 function Breadcrumbs() {
   const { contract: currentContract, key } = Route.useParams()
-  const { data: contracts } = useContracts()
+  const contracts = Route.useLoaderData()
```

Comment:
I was getting a really odd "bad setState" error here. The combination of calling `useContracts` (which calls `useSuspenseQuery` under the hood) and calling `navigate` was causing a React error. Using the loader data fixed it. It feels like there's a bug in either TanStack Router or TanStack Query.